cmd/crc/cmd/start.go

```diff
@@ -53,11 +53,14 @@ func runStart(arguments []string) {
 	}
 
 	commandResult, err := machine.Start(startConfig)
-	logging.InfoF(commandResult.Status)
 	if err != nil {
-		errors.ExitWithMessage(1, err.Error())
+		errors.Exit(1)
```

Comment:
Are we not putting the exit message along with err, when the err is not nil?

Reply:
On error creation, we already had the error message printed, so it does not need to be printed again.